File: python/tmdb/tmdb.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

from __future__ import (division, absolute_import, print_function,
                        unicode_literals)

import os
import requests

api_key = os.environ['TMDB_API']

response = requests.get('https://api.themoviedb.org/3/discover/movie?api_key=' +
                        api_key + '&primary_release_year=2017&sort_by=revenue.desc')

print(response.json())


def main():
    pass


if __name__ == '__main__':
    main()

File: go/tmdb/tmdb.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"os"
)

func main() {

	var apikey = os.Getenv("TMDB_API")

	response, err := http.Get("https://api.themoviedb.org/3/discover/movie?api_key=" + apikey + "&primary_release_year=2017&sort_by=revenue.desc") // pragma: allowlist secret

	if err != nil {
		fmt.Printf("The HTTP request failed with error %s\n", err)
	} else {
		data, _ := ioutil.ReadAll(response.Body)
		fmt.Println(string(data))
	}

}
